fhir-server/liberty-config-tenants/config/tenant1/fhir-server-config.json

```diff
@@ -50,6 +50,14 @@
                     "currentSchema": "APP"
                 }
             }
+        },
+        "operations": {
```

Comment:
documentation for new config props should go in FHIR Server User's Guide section 5

Reply:
Docs are updated...